package.json

```diff
@@ -167,12 +172,7 @@
     "webpack-dev-server": "^4.1.1",
     "webpack-manifest-plugin": "2.0.4"
   },
-  "overrides": {
-    "chokidar": "^3.4.1",
-    "watchpack": "^2.4.0",
-    "webpack": {
-      "watchpack": "^2.4.0"
-    },
-    "loader-utils": "^2.0.0"
+  "resolutions": {
```

Comment:
mirroring internal changes, npm <=7 does not support "overrides" key, needs to use resolutions instead